onnx/version_converter/adapters/prelu_7_6.h

```diff
@@ -0,0 +1,28 @@
+// Adapter for PRelu in default domain from version 8 to 7
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+class PRelu_7_6 final : public Adapter {
+  public:
+    explicit PRelu_7_6(): Adapter("PRelu", OpSetID(7), OpSetID(6)) {}
+
+    void adapt_prelu_7_6(std::shared_ptr<Graph> graph, Node* node) const {
+      // Throw an exception if any broadcasting occurs
+      const ArrayRef<Value*>& inputs = node->inputs();
+      std::vector<Dimension> X_sizes = inputs[0]->sizes();
+      std::vector<Dimension> slope_sizes = inputs[1]->sizes();
```

Comment:
It's possible, we don't have the shape information. If there is no enough information, also fail the conversion.